homeassistant/components/tradfri/.translations/sv.json

```diff
@@ -1,8 +1,8 @@
 {
     "config": {
         "abort": {
-            "already_configured": "Bryggan \u00e4r redan konfigurerad",
-            "already_in_progress": "Konfigurations fl\u00f6det f\u00f6r bryggan p\u00e5g\u00e5r redan."
+            "already_configured": "Gatewayen \u00e4r redan konfigurerad",
```

Comment:
Translation changes are handled in Lokalise:
https://developers.home-assistant.io/docs/en/internationalization_backend_localization.html#translation-strings